templates/static/js/example-tools.js

```diff
@@ -0,0 +1,64 @@
+(function () {
+  function inIframe() {
+    try {
+      return window.self !== window.top;
+    } catch (e) {
+      return true;
+    }
+  }
+
+  function fragmentFromString(htmlString) {
+    var temp = document.createElement('template');
+    temp.innerHTML = htmlString;
+    return temp.content;
+  }
+
+  if (!inIframe()) {
+    document.documentElement.classList.add('u-baseline-grid');
+
+    document.addEventListener('DOMContentLoaded', function () {
+      var body = document.body;
+      var controls = fragmentFromString(
+        '<div class="u-baseline-grid__toggle"><label class="p-switch"><input type="checkbox" class="p-switch__input js-baseline-toggle" /><span class="p-switch__slider"></span><span class="p-switch__label">Toggle baseline grid</span></label></div>',
+      );
+      var themes = fragmentFromString(
+        '<div class="u-theme-toggle"><button class="u-theme-toggle__button u-theme-toggle__dark js-dark-theme-toggle">Dark</button><button class="u-theme-toggle__button u-theme-toggle__light js-light-theme-toggle">Light</button><button class="u-theme-toggle__button u-theme-toggle__paper js-paper-theme-toggle">Paper</button></div>',
+      );
+
+      body.appendChild(themes);
+      body.appendChild(controls);
+
+      var toggle = document.querySelector('.js-baseline-toggle');
+      toggle.addEventListener('click', function (event) {
+        body.classList.toggle('u-baseline-grid');
+      });
+
+      // var is_not_themed = document.getElementById('is-not-themed').getAttribute('data-is_not_themed');
+
+      var themeToggles = document.querySelector('.u-theme-toggle');
+
+      !show_theme_switch && themeToggles.classList.add('u-hide');
```

Comment:
While this works (and is short oneliner) it's easier to read as `if (!SHOW_THEME_SWITCH) { … }`.
We can leave optimisations to the compiler or compressor.